src/video_core/shader/generator/glsl_shader_decompiler.cpp

```diff
@@ -12,9 +12,9 @@
 #include <nihstro/shader_bytecode.h>
 #include "common/assert.h"
 #include "common/common_types.h"
-#include "video_core/renderer_opengl/gl_shader_decompiler.h"
+#include "video_core/shader/generator/glsl_shader_decompiler.h"
 
-namespace OpenGL::ShaderDecompiler {
+namespace Pica::Shader::Generator {
```

Comment:
```suggestion
namespace Pica::Shader::GlslGenerator {
```
in case we do SPIRV in the future

Reply:
> in case we do SPIRV in the future

I think it's fine since the cpp file name already stated as GLSL

Reply:
```Pica::Shader::Generator``` contains all the generators including the SPIR-V fragment generator, not just GLSL.

Reply:
> `Pica::Shader::Generator` contains all the generators including the SPIR-V fragment generator, not just GLSL.

In that case the cpp file name should be renamed to avoid confusion?

Reply:
I've moved some of the common shader config code out of the GLSL generator and moved both the GLSL and SPIR-V generators into their own sub-namespaces to address this.